README.md

```diff
@@ -1 +1,63 @@
-# PlyIO
+# Ply polygon file IO
+
+**PlyIO** is a package for reading and writing data in the
+[Ply](http://paulbourke.net/dataformats/ply/) polygon file format, also called
+the Stanford triangle format.
+
+## Quick start
+
+Here's an example of how to write a basic ply file containing random triangles
+and edges:
+
+```julia
+using PlyIO
+
+ply = Ply()
+push!(ply, PlyComment("An example ply file"))
+
+nverts = 1000
+
+# Random vertices with position and color
+vertex = PlyElement("vertex",
```

Comment:
Ok I shortened all the names and put them into a `Ply` submodule on the branch.  It's not clear which is better, see:

https://github.com/FugroRoames/PlyIO.jl/tree/api-experiment